crates/biome_js_analyze/src/semantic_analyzers/nursery/no_empty_type_parameters.rs

```diff
@@ -0,0 +1,84 @@
+use crate::semantic_services::Semantic;
```

Comment:
The rule should be placed in `biome_js_analyze/src/syntax/nursery/` because it is a syntax rule. Syntax rule are always checked (even when the linter is disabled).

Reply:
I am sorry, I was definitely wrong here. It is not a syntax rule because i this not a parse error. I will fix the issue.